jq.js

```diff
@@ -1,38 +1,38 @@
-loadScript("jq-emscripten.js");
+this.jq = this.require("./jq-emscripten") || this.jq;
 
-(function() {
-    var ProcessWithJQ;
-    ProcessWithJQ = function() {
-        var jqOutput = '';
-        this.evaluate = function() {
-            jq(['-M', this.jqargs, '/data/inputfile'], this.jsonInput, function(jqResult) {
-                jqOutput = jqResult
-            });
-            if (jqOutput.match(/^jq: error:/)) {
-                console.error(jqOutput);
-                return null;
-            } else {
-                return jqOutput;
-            }
-        };
-        this.title = function() {
-            return "jq JSON processor";
-        };
-        this.text = function() {
-            return null;
-        };
-    };
+var ProcessWithJQ = function() {
+  var flags = '-M';
 
-    ProcessWithJQ.identifier = "com.virtzilla.PawExtensions.jq";
-    ProcessWithJQ.title = "jq JSON processor";
-    ProcessWithJQ.help = "https://stedolan.github.io/jq/manual/#Basicfilters"
-    ProcessWithJQ.inputs = [
-        DynamicValueInput("jqargs", "JQ args", "String", {
-            defaultValue: "."
-        }),
-        DynamicValueInput("jsonInput", "JSON input", "String", {
-            placeholder: "{\"change me\": \"put your JSON input here\"}"
-        })
-    ];
-    registerDynamicValueClass(ProcessWithJQ);
-}).call(this);
+  this.evaluate = function() {
+    if (this.rawOutput)
+      flags += 'r';
+
+    var jqOutput = jq.raw(this.jsonInput, [flags, this.jqargs]);
```

Comment:
Hi @estum great work!
I was desperately needing this feature, and went ahead and copied the code into my local plugin. 
Turns our there's a bug in this line. 
I actually created an issue on https://github.com/luckymarmot/Paw-Issues/issues/104 because of it. 
The issue is that jq.raw() is returning an extra `\n` in the end. 
I added a suggestion to the code that fixes the issue on my side.